tinyagentos/routes/activity.py

```
@@ -1,116 +1,560 @@
-            "vram_used_mb": vram_used_mb,
-            "vram_total_mb": vram_total_mb,
-            "type": gpu_name,
-        },
-        "thermal": get_thermal_zones(),
-        "zram": get_zram_stats(),
-        "disk": disk_info,
-        "network": get_network_rates(),
-        "processes": get_top_processes(limit=10),
-    })
+        body = await request.json()
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)